manage_user_edit_page.php

```diff
@@ -127,6 +127,24 @@
 				</td>
 			</tr>
 
+			<!-- Password -->
+			<?php
+				if ( MD5 == $g_login_method ) {
```

Comment:
We shouldn't have password type specific logic here.  Look at account_update.php and see how it changes the password via standard APIs that handles all hashing models via `user_set_password()`.